### DISCUSSION TITLE
Linux/arm64 platform support

### DISCUSSION THREAD

**Discussion by mystique09** [livebud/bud#103] [Proposals]
I tried running the installer in Termux, please add termux support XD.

```fish
livebud/bud: platform linux/arm64 is not supported.  Make sure this script is up-to-date and file request at https://github.com/livebud/bud/issues/new
```

**Comment by matthewmueller** [comment]
Thanks @mystique09, this should be pretty simple. 

Changes needed:
- Update Makefile release script to include linux/arm64
- Update install script
- Update CI to test this platform

Here's an example of getting it working with darwin/arm64:
- https://github.com/livebud/bud/commit/00b5fae20e9ca1fb2f5ff3fa72912e3460fd9d7c
- https://github.com/livebud/bud/commit/6669144024f607f8838905078313e817401f0852

Open to creating a PR?

**Reply by mystique09** [reply]
Sure, wait

**Comment by matthewmueller** [comment]
_(Converting to a discussion since this is currently working as expected)_
